src/Explorer/Explorer.tsx

```diff
@@ -875,9 +924,26 @@
       handleError(error, "Explorer/onNewNotebookClicked");
       throw new Error(error);
     }
+    if (userContext.features.notebooksTemporarilyDown === false && userContext.features.phoenix === true) {
```

Comment:
>(userContext.features.notebooksTemporarilyDown === false && userContext.features.phoenix === true [](http://example.com/codeflow?start=7&length=97)

this is in multiple places.. may be good to have this chk as one function so it is easy to update

Reply:
Added function